fabfile/setup.py

```diff
@@ -43,7 +43,7 @@ def create_ansible_env():
             local('mkvirtualenv ansible')
     with ansible():
         with settings(warn_only=True):
-            local('pip install ansible')
+            local('pip install ansible==1.9.4')
```

Comment:
if we don't have one, we should make an issue for supporting Ansible 2 - I think it's just our `django_environments` stuff that's keeping it from working